### DISCUSSION TITLE
Crystallography martensite calculator

### DISCUSSION THREAD

**Discussion by mugikumar** [ORTools4MTEX/ORTools#35] [Q&A]
Hi all,
      I doubt selecting the stress state in this code.
Sig.labels = {'RD_T','RD_PS','BB','TD_PS','TD_T','TD_DD','TD_S','RD_EX','RD_C','RD_SH','BB_C','TD_SH','TD_C','TD_EX','RD_S','RD_DD'};
Sig.choice = [  1       1      1     1      1       1      1       1      1       1      1       1      1       1      1       1];
if materials are in ausformed (hot rolling), how to select the stress state?

**Comment by frankNiessen** [comment]
Hi,

I assume you're referring to [this](https://github.com/frankNiessen/martensiteWorkCalculator/blob/master/martensiteWorkCalculator.m) repository (which is not part of ORTools, so better to ask at this repository in the future). 
So these idealised stress states are defined in the function [cmptSigma](https://github.com/frankNiessen/martensiteWorkCalculator/blob/master/src/cmptSigma.m). You will have to decide for your individual case which stress state best represents your actual processing conditions. You are also able to add your own complex stress state to this function or define it directly in the work calculator.

Best wishes
Frank
